src/core/utils/transactions.ts

```diff
@@ -173,6 +173,19 @@ const getAssetFromChanges = (
   return changes[0]?.asset;
 };
 
+const getAddressToFromChanges = (
+  changes: {
+    direction: TransactionDirection;
+    asset: ParsedUserAsset;
+    address_to: Address;
+  }[],
+  type: TransactionType,
+) => {
+  if (type === 'sale')
+    return changes?.find((c) => c?.direction === 'out')?.address_to;
+  return changes[0]?.address_to;
+};
```

Comment:
Found one edge case with an Approval where the user address is still displayed

Reply:
i am not seeing a to field at all on older approvals